dashboard/templates/dashboard/includes/user_profile_post_comments.html

```diff
@@ -0,0 +1,48 @@
+{% load staticfiles %}
+<div class="sd-profile-comments-title">
+    <h5>Comments</h5>
+</div>
+<div id="sd-profile-comments-wrapper">
+        {% if user_posts %}
+            <div class="media-list">
+                {% for post in user_posts %}
```

Comment:
That's correct. I should I have explained a little bit. Since Alanna had worked on the comments I was leaving it her to carry on from there. Essentially the posts were used as dummy content.